frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -1071,12 +1071,6 @@ describe("scenarios > question > filter", () => {
           type: "query",
         },
       });
-
-      cy.findByRole("link", { name: "Sample Dataset" })
```

Comment:
I removed this assertion because I think it's the spirit of the breadcrumb (to display e.g. filtered category).

Anyway, if we want to remove this kind of feature completely, that's for another time and probably requires consulting the product team first.